client/src/models/Location.js

```diff
@@ -83,6 +101,15 @@ export default class Location extends Model {
 
   static autocompleteQuery = "uuid, name"
 
+  static parseCoordinate(latLng) {
+    const value = parseFloat(latLng)
+    if (!value && value !== 0) {
+      return null
+    }
+    // 4 decimal point precision seems precise enough. https://stackoverflow.com/a/16743805/1209097
+    return parseFloat(value.toPrecision(6))
```

Comment:
The comment and the code are not in 100% agreement; `.toPrecision(6)` could, with WGS84 coordinates, have anywhere between 3 (for `123.123`) and 6 (for `0.123456`) decimals. Did you mean to use `.toFixed(4)`? Note: 4 decimals seems too little to me; with 6 decimals you get about one 10 cm precision, with 4 it's 10 meters. Perhaps @vassil can shed some light on how precise it needs to be.
Also note that e.g. https://github.com/NCI-Agency/anet/pull/2943/files#diff-4ff18ae18c29280dbc33bc37e9a977d1R78 should be in accordance with this if we want to be consistent throughout the UI.

Reply:
You are right, it was meant to be `toFixed(4)`. i changed it that way. `Coordinate` component was displaying 3 decimals before. That seemed too little to me, that's why I made it 4 decimals (10 meters precision). I'd like to hear @VassilIordanov thinks about this.

Reply:
I think 10 meters is probably slightly on the not enough side. It could be the difference between 2 buildings. I suggest 6 digits